aderyn_core/src/detect/high/_template.rs

```diff
@@ -22,6 +22,7 @@ pub struct TemplateDetector {
     // Keys are: [0] source file name, [1] line number, [2] character location of node.
     // Do not add items manually, use `capture!` to add nodes to this BTreeMap.
     found_instances: BTreeMap<(String, usize, String), NodeID>,
+    hints: BTreeMap<(String, usize, String), String>,
 }
```

Comment:
Key is the same as found_instances but the value refers to to the actual hint associated to the instance